src/containters/Home/components/HeroView/index.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import './index.scss';
+import { IoIosArrowForward } from 'react-icons/io';
+import { BrowserRouter as Router, Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+const HeroView = (props) => {
```

Comment:
- Please add 1 empty line between file imports and component definition for readability.